src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -143,9 +143,9 @@ internal class PicklingSqlCommandExecutor : SqlCommandExecutor
                         // The following can happen if an UDF takes Row object(s).
                         // The JVM Spark side sends a Row object that wraps all the columns used
                         // in the UDF, thus, it is normalized below (the extra layer is removed).
```

Comment:
Is this comment still relevant?

Reply:
Worker will crash without this, so I believe it is ?

Reply:
oh I meant respect to the code. I think "extra layer is removed" is regarding the `RowConstructor`, but now that it's gone, is the comment up to date?

Reply:
Extra layer can refer to `Row`, so we take out `Values` from it ?

Reply:
I'm okay with removing the ( )'s though if things sound unclear.

Reply:
@elvaliuliuliu do we need to update the description or does it still apply ?